formidling/src/main/java/no/nav/ung/sak/formidling/vedtak/DetaljertResultatType.java

```diff
@@ -1,17 +1,24 @@
 package no.nav.ung.sak.formidling.vedtak;
 
 public enum DetaljertResultatType {
-    INNVILGET_NY_PERIODE("Innvilget ny periode"),
     AVSLAG_INNGANGSVILKÅR("Avslag inngangsvilkår"),
-    AVSLAG_UTTAK("Avslag uttak"),
     ENDRING_ØKT_SATS("Endring økt sats 25 prosent"),
     ENDRING_RAPPORTERT_INNTEKT("Reduksjon pga inntekt"),
-    AVSLAG_RAPPORTERT_INNTEKT("Avslag pga rapportert inntekt")
+    AVSLAG_RAPPORTERT_INNTEKT("Avslag pga rapportert inntekt"),
+    IKKE_VURDERT("Ikke vurdert"),
+    INNVILGELSE_VILKÅR_NY_PERIODE("Innvilgelse av vilkår for ny periode uten utbetaling"),
+    INNVILGELSE_UTBETALING_NY_PERIODE("Innvilgelse av ny periode med utbetaling"),
```

Comment:
Kvifor må vi skilje på desse?

Reply:
For å skille mellom innvilgelse med og uten utbetaling. I førstegangsbehandling finnes det begge deler. 

Foreløpig bruker jeg det ikke, men jeg tror det kan være interessant å vise det som et resultat. Tror du ikke det? Kan evt. innføre det når det blir behov.

Reply:
Tenkjer du no for brev eller frontend?